lib/mongodb-scope-client-mixin.js

```diff
@@ -0,0 +1,40 @@
+var result = require('lodash.result');
```

Comment:
Ooof... embarrassing. Pretty sure this can all go away now. 

`<aside>`
The data-service router call: I started mongodb-js/data-service#139 to rip out the long deprecated `router` class references (among other old junk) awhile back but had to switch to other things.